### ISSUE TITLE
Fails to parse node js config if key value is empty

### ISSUE THREAD

**Issue by flopacero** [newrelic/newrelic-diagnostics-cli#201]
NR diagnostic CLI fails at the beginning of the run with a message:

```
goroutine 20 [running]:
github.com/newrelic/newrelic-diagnostics-cli/tasks/base/config.parseJs({0x8bf800, 0xc00018f6c8})
        /home/runner/work/newrelic-diagnostics-cli/newrelic-diagnostics-cli/tasks/base/config/validate.go:373 +0x1365
github.com/newrelic/newrelic-diagnostics-cli/tasks/base/config.processConfig({{0xc00038d468?, 0x1?}, {0xc00002d5c0?, 0xc000692a00?}})
        /home/runner/work/newrelic-diagnostics-cli/newrelic-diagnostics-cli/tasks/base/config/validate.go:187 +0x33d
github.com/newrelic/newrelic-diagnostics-cli/tasks/base/config.BaseConfigValidate.Execute({}, {0x414025?}, 0xc000498000?)
        /home/runner/work/newrelic-diagnostics-cli/newrelic-diagnostics-cli/tasks/base/config/validate.go:102 +0x249
main.processTasks({0xc00025ac00?}, {0x0, 0x0, 0x0?}, 0xc0001a51d0?)
        /home/runner/work/newrelic-diagnostics-cli/newrelic-diagnostics-cli/processTasks.go:121 +0xa14
created by main.main
```

It likely happens due to a `tasks/base/config/validate.go:373` where it has a line
```go
if string(strings.TrimSpace(keyMap[1])[0]) == "[" {
```

there is an assumption that if key-value string is split by colon then after a trim a second part is non-empty string. This fails for a string like 
```
agent_enabled: 
   process.env.NODE_ENV !== 'test'
```
due to a line break which causes OutOfBound error above.

**Comment by cade-conklin** [comment]
Hi @flopacero!  Thank you for bringing this to our attention.  We will add this to our stream of work!

**Comment by daffinito** [comment]
Hi @flopacero, thanks again for reporting this! We just released version 3.2.3 of the Diagnostic CLI which resolves the panic. Let us know if you run into any further issues!
